src/configurePipeline/model/Common.ts

```diff
@@ -0,0 +1,61 @@
+import { ServiceClientCredentials } from 'ms-rest';
```

Comment:
I think we are following fileName pattern starting with small case so we should use that everywhere -> common.ts